### DISCUSSION TITLE
Add note in addon Help/Readme about this Discussion area

### DISCUSSION THREAD

**Discussion by TonyGravagno** [RealRaven2000/FiltaQuilla#284] [Ideas]
Community only happens if people know about it. If this forum is accepted as a useful resource, I recommend adding info to guide people here from the various addon text/help resources in Add-ons Manager, addons.thunderbird.net, etc.

Invite people here whenever the opportunity arises. Get fellow users involved in the discussion. Some people are afraid of "Issues" and "Tickets" but they have no problem creating or adding to a thread in a forum like this.

Frankly, not only is this a resource for information and collaboration, it's also a marketing tool which helps to reassure people that the software is vibrant and that the author is responsive. We here already see this in tickets and software updates. But most users don't go into tickets, and they auto-update, so they have no idea what's happening out here.

And if that's not clear enough ... 😁 ... good marketing leads to more licenses as more people get familiar with the ecosystem. And, uh, and **that** means \$£€ $money$ for Axel, which helps to keep him motivated and thus helps to keep this fine software free/libre and high quality for all of us. All of this stuff is linked!

**Comment by RealRaven2000** [comment]
> Frankly, not only is this a resource for information and collaboration, it's also a marketing tool which helps to reassure people that the software is vibrant and that the author is responsive. We here already see this in tickets and software updates. But most users don't go into tickets, and they auto-update, so they have no idea what's happening out here.

most users do not even read the change log, and I translate into up to 21 languages. It;s actually difficult to find a prominent point for this information but I am open to suggestions. How about settings / Help & Support page?

**Reply by TonyGravagno** [reply]
Yeah, the idea is to bring people into the fold. Even if they read nothing else, they should know where to go to ask for help or to talk about their software. From that contact point they can be funneled to other information and ideally a forum that suits their preferences.

(kind of exaggerating here) I think any product would do well to have an in-product landing page with nothing more that two buttons : Information and Contact. If you want any kind of information, go here. If you want to contact anyone about anything, go there. Everything after that about features, change logs, roadmaps, reviews, etc is nice for those of us who care but not at all relevant to the other 98.2%. If someone isn't inclined to click one of those two buttons, they aren't inclined to read anything that follows.

**Reply by TonyGravagno** [reply]
OOPS

> How about settings / Help & Support page?

Yes, I believe so.

**Comment by RealRaven2000** [comment]
> And if that's not clear enough ... 😁 ... good marketing leads to more licenses as more people get familiar with the ecosystem. And, uh, and **that** means $£€

we need to be careful I was told my other add-ons are not libre because they use the wrong type of license. I may well change it to LPG3 even if it is slightly risky as the discussion about it may be more tiring than living with the uncertainty of an income. simply not sure

**Reply by TonyGravagno** [reply]
I might not understand the terms of addon licensing in this Thunderbird ecosystem yet. I assumed it was like every other platform with a freemium model. Let's pick that up in a separate thread.

My sub-point on that was ultimately that Marketing (evangelism, fandom, word of mouth, promotion) serves everyone in that the more people who use software (even FOSS by any license) the more chances there are that some will motivate the developer(s) to continue providing it for all. And **that** is one of my reasons for recommending notifications about this and other communications resources.
